src/storage/box_storage.rs

```diff
@@ -0,0 +1,100 @@
+use alloc::boxed::Box;
+
+use atomic::decompose_data;
+
+use {AtomicTmpl, OwnedTmpl, SharedTmpl, Pointer, Storage};
+
+/// An atomic pointer that can be safely shared between threads.
+///
+/// See [`AtomicTmpl`] for more details.
+///
+/// [`AtomicTmpl`]: struct.AtomicTmpl.html
+pub type Atomic<T> = AtomicTmpl<T, Box<T>>;
```

Comment:
I wonder if it's a breaking change.  `Atomic` is renamed into `AtomicTmpl`, with an additional type parameter, and `Atomic` is now defined as a type alias.